src/style.css

```
@@ -1 +1,68 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

```suggestion
  box-sizing: border-box;
```
Don't reset margins using * selector. Better specify the exact tags you want to reset.
You can also set the font for the whole page and inherit where needed

---

Я не розумію, що саме ви маєте на увазі. Можа будь ласка приклад

---

You can reset margin and paddings for exact tags as follow:
```
html, 
body, 
p {
  margin: 0;
  padding: 0;
}
```